src/data/finsupp.lean

```diff
@@ -1751,4 +1751,32 @@ lemma swap_mem_antidiagonal_support {n : σ →₀ ℕ} {f} (hf : f ∈ (antidia
   f.swap ∈ (antidiagonal n).support :=
 by simpa only [mem_antidiagonal_support, add_comm, prod.swap] using hf
 
+lemma finite_le_nat (n : σ →₀ ℕ) : set.finite {m | m ≤ n} :=
```

Comment:
Maybe add a docstring (the sole statement `set.finite {m | m ≤ n}` can be misleading, I thought at first it had to do with `set.range n`)

Reply:
Done.